lib/run-script.js

```diff
@@ -7,7 +7,7 @@ var lifecycle = require("./utils/lifecycle.js")
   , log = require("npmlog")
   , chain = require("slide").chain
 
-runScript.usage = "npm run-script <command> [-- <args>]"
+runScript.usage = "npm run-script <command>|env [-- <args>]"
```

Comment:
good point. i recall trying to find 'test' called out and failing... not sure why i left this here.